components/eamxx/src/physics/rrtmgp/scream_rrtmgp_interface.cpp

```diff
@@ -760,6 +820,9 @@ namespace scream {
             // Combine gas and cloud optics
             clouds_day.delta_scale();
             clouds_day.increment(optics);
+            if (extra_clnsky_diag) {
+                clouds_day.increment(optics_no_aerosols);
```

Comment:
I'm not familiar with the `OpticalProps2str` data structure, so I can't comment. I'll let Ben decide.

Reply:
My initial reaction to this is that I don't personally like the idea of adding a decrement method to the external optics class, but I'll have to think about that a little more. I think it makes more sense to create the combinations we want here by addition rather than subtraction, and in the future based on a set of lists of constituents for each diag call.

Reply:
I'm not really following. If/when you want to implement this, you can of course ping me and go over the details, if you need.

Reply:
Naser and I talked about this a week or two ago, I actually have a slightly different idea about how to handle this.